applications/TopologyOptimizationApplication/custom_utilities/filter_function.h

```diff
@@ -0,0 +1,256 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Baumgärtner Daniel, https://github.com/dbaumgaertner
+//                   Octaviano Malfavón Farías
+//                   Eric Gonzales
+//					 Philipp Hofer
+//					 Erich Wehrle
+//
+// ==============================================================================
+
+#if !defined(KRATOS_FILTER_FUNCTION_H_INCLUDED)
+#define  KRATOS_FILTER_FUNCTION_H_INCLUDED
+
+// System includes
+
+// External includes
+
+// Project includes
+
+
+namespace Kratos
+{
+
+///@name Kratos Globals
+///@{
+
+///@}
+///@name Type Definitions
+///@{
+
+
+///@}
+///@name  Enum's
+///@{
+
+///@}
+///@name  Functions
+///@{
+
+///@}
+///@name Kratos Classes
+///@{
+
+/// Solution utility to filter results.
+/** Detail class definition.
+
+ */
+
+class FilterFunction
+{
+public:
+
+    ///@name Type Definitions
+    ///@{
+
+    /// Pointer definition of FilterFunction
+    KRATOS_CLASS_POINTER_DEFINITION(FilterFunction);
+
+    ///@}
+    ///@name Life Cycle
+    ///@{
+
+    /// Default constructor.
+    FilterFunction( std::string FilterFunctionType, const double SearchRadius )
```

Comment:
I'd make this a static function.

Reply:
How can I do that? Defining it in the constructor?